.github/workflows/lint.yml

```
@@ -4,7 +4,7 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

I would put 24.04 just in case
```suggestion
    runs-on: ubuntu-24.04
```